src/OpenTelemetry/Trace/ActivityExporter.cs

```diff
@@ -75,16 +78,18 @@ public void Shutdown(int timeoutMilliseconds = Timeout.Infinite)
 
             if (Interlocked.Increment(ref this.shutdownCount) > 1)
             {
-                return; // shutdown already called
+                return false; // shutdown already called
             }
 
             try
             {
                 this.OnShutdown(timeoutMilliseconds);
+                return true; // TODO: update exporter.OnShutdown to return boolean
```

Comment:
Put this `return true` to the end of this function (right before `}`)?

Reply:
The way I look at `try { return } catch() { return }` is similar to `if() { return } else { return }`. Do you see some benefit of putting it outside?

Reply:
I thought normal execution flow would end at `}`, besides that, I don't see any advantage to move `return true` to the end of the function.